workflows/differentialabundance.nf

```diff
@@ -400,7 +400,7 @@ workflow DIFFERENTIALABUNDANCE {
     }
 
     // We'll use a local module to filter the differential tables and create output files that contain only differential features
-    ch_logfc = Channel.value([ params.differential_fc_column, params.differential_min_fold_change ])
+    ch_logfc = Channel.value([ params.differential_fc_column, Math.log(params.differential_min_fold_change)/Math.log(2) ])
```

Comment:
I'm not sure what you're doing here, but could you do it in the module?

Reply:
Can do